doc/scripts/launch_hard_rt_vm.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# Copyright (C) 2019 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+
+#!/bin/bash
```

Comment:
Is this repeat needed (the first line is the same)?

Reply:
Fixed.